dotcom-rendering/src/web/components/TweetBlockComponent.tsx

```diff
@@ -13,6 +13,10 @@ const noJSStyling = css`
 		${body.small()};
 	}
 
+	.twitter-tweet iframe {
+		width: 100% !important;
```

Comment:
Unfortunate that we need to use !important here but I don't think theres any other way we overwrite Twitters CSS.

Reply:
If this works I'll take it! Could we add a comment here clarify why we use `!important` maybe?

Reply:
I think you might need to add something like [this](https://github.com/guardian/dotcom-rendering/blob/bd23511a016767ee2d24cce03573fb1e76479710/dotcom-rendering/src/web/components/SignInGate/gateDesigns/shared.tsx#L65) because of our linting.